targets/tizenrt-artik053/configs/jerryscript/defconfig

```diff
@@ -250,47 +204,53 @@ CONFIG_BOOT_RUNFROMFLASH=y
 # Boot Memory Configuration
 #
 CONFIG_RAM_START=0x02023800
-CONFIG_RAM_SIZE=804864
+CONFIG_RAM_SIZE=968704
+# CONFIG_DDR is not set
 # CONFIG_ARCH_HAVE_SDRAM is not set
 
 #
 # Board Selection
 #
 CONFIG_ARCH_BOARD_ARTIK053=y
+# CONFIG_ARCH_BOARD_ARTIK053S is not set
+# CONFIG_ARCH_BOARD_ARTIK055S is not set
 # CONFIG_ARCH_BOARD_SIDK_S5JT200 is not set
-CONFIG_ARCH_BOARD="artik053"
+CONFIG_ARCH_BOARD_ARTIK05X_FAMILY=y
+CONFIG_ARCH_BOARD="artik05x"
 
 #
 # Common Board Options
 #
 # CONFIG_BOARD_CRASHDUMP is not set
+# CONFIG_BOARD_ASSERT_AUTORESET is not set
 CONFIG_LIB_BOARDCTL=y
 CONFIG_BOARDCTL_RESET=y
 # CONFIG_BOARDCTL_UNIQUEID is not set
-# CONFIG_BOARD_COREDUMP_FLASH is not set
 # CONFIG_BOARD_FOTA_SUPPORT is not set
-# CONFIG_BOARD_RAMDUMP_FLASH is not set
-# CONFIG_BOARD_RAMDUMP_UART is not set
 
 #
 # Board-Specific Options
 #
-CONFIG_ARTIK053_BOOT_FAILURE_DETECTION=y
-CONFIG_ARTIK053_BOOT_COUNTS_ADDR=0x80090810
-CONFIG_ARTIK053_FLASH_CAPACITY=8388608
-CONFIG_ARTIK053_FLASH_PAGE_SIZE=4096
-CONFIG_ARTIK053_FLASH_PART=y
-CONFIG_ARTIK053_FLASH_MINOR=0
-CONFIG_ARTIK053_FLASH_PART_LIST="16,48,192,32,512,2400,1536,1536,1000,400,8,512,"
-CONFIG_ARTIK053_FLASH_PART_TYPE="none,ftl,none,none,none,none,none,ftl,smartfs,romfs,config,none,"
-CONFIG_ARTIK053_FLASH_PART_NAME="bl1,sssro,bl2,sssfw,wlanfw,os,factory,ota,user,rom,nvram,sssrw,"
-CONFIG_ARTIK053_AUTOMOUNT=y
-CONFIG_ARTIK053_AUTOMOUNT_USERFS=y
-CONFIG_ARTIK053_AUTOMOUNT_USERFS_DEVNAME="/dev/smart0p8"
-CONFIG_ARTIK053_AUTOMOUNT_USERFS_MOUNTPOINT="/mnt"
-
-#
-# RTOS Features
+CONFIG_ARTIK05X_BOOT_FAILURE_DETECTION=y
+CONFIG_ARTIK05X_BOOT_COUNTS_ADDR=0x80090810
+CONFIG_ARTIK05X_FLASH_CAPACITY=8388608
+CONFIG_ARTIK05X_FLASH_PAGE_SIZE=4096
+CONFIG_ARTIK05X_FLASH_PART=y
+CONFIG_ARTIK05X_FLASH_MINOR=0
+CONFIG_ARTIK05X_FLASH_PART_LIST="16,48,192,32,512,2400,1536,1536,200,1200,8,512,"
```

Comment:
Same here. Summon @rtakacs .

Reply:
If **CONFIG_ROMFS** is enabled, there must be a new **ROM** entry in the partition list (otherwise you can't flash the device). I've introduced a new 1200 KB memory area for that. The **SMARTFS** (user rw space) is decreased to 200 KB since JerryScript cannot writes files (so it's unnecessary to have 1000 KB **SMARTFS**).